tests/reftests/run.ml

```diff
@@ -476,6 +484,65 @@ let run_test ?(vars=[]) ~opam t =
           List.fold_left
             (fun vars (v, r) -> (v, r) :: List.filter (fun (w, _) -> v <> w) vars)
             vars bindings
+        | Cat files ->
+          let print_opamfile header file =
```

Comment:
This probably deserves to be a top-level function :grin:

Reply:
it will be removed from `run.ml` to be an outside program